src/viz/validation.ts

```diff
@@ -50,19 +50,19 @@ No files have been created.
 };
 
 export const parseBucketName = (bucketPath: string): string | undefined => {
-  const matches = bucketPath.match(/(gs:\/\/[^/\s]+)/);
+  const matches = bucketPath.match(/(gs:\/\/[-a-zA-Z]+)\/?([-a-zA-Z]+\/?)*$/);
   if (matches === null) {
     return undefined;
   }
-  return matches[0];
+  return matches[1];
```

Comment:
Revert this if you accept the regex changes up above

```suggestion
  return matches[0];
```

Reply:
It actually should have been 1 the whole time. The return from `.matches` in javascript always has the entire string as `[0]`, then groups start after that.